core/server/models/base/index.js

```diff
@@ -519,8 +528,8 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
      * @param {Object} data Has keys representing the model's attributes/fields in the database.
      * @return {Object} The filtered results of the passed in data, containing only what's allowed in the schema.
      */
-    filterData: function filterData(data) {
-        var permittedAttributes = this.prototype.permittedAttributes(),
+    filterData: function filterData(data, {methodName, context} = {}) {
```

Comment:
I made this function signature like this because you mentioned before you didn't like functions with too many arguments.

Reply:
> you didn't like functions with too many arguments.

We follow the concept of `(data, options)`. That looks fine 👍